docs/node_monitoring.md

```diff
@@ -327,7 +327,7 @@ Please click the `Plus`-button in the main navigation and select `import`.
   <img src={useBaseUrl('/node-monitoring/grafana-import.png')} />
 </div>  
 
-We will use the [HydraDX Dashboard](https://grafana.com/grafana/dashboards/14158) and to load it you simply input the id `14158` and hit the `Load`-button.  
+We will use the [HydraDX Dashboard](https://grafana.com/grafana/dashboards/14158) layout, so simply input the id `14158` and hit the `Load`-button.  
```

Comment:
this change is debatable. could be removed. But the first time I read this line, it sounded like my collator node metrics were going to show up on a general HydraDX Dashboard of all nodes. So I added the "layout" word to maybe communicate it was just a visual setup.